src/environment/stella_environment.cpp

```diff
@@ -212,7 +212,11 @@ const ALEState& StellaEnvironment::getState() const {
 }
 
 void StellaEnvironment::saveScreenPNG(const string& filename) {
-  m_osystem->p_export_screen->save_png(m_screen, filename);
+
```

Comment:
This seems inefficient. We should have stella env maintain a ScreenExporter as a private member. This member could also be used when fulfilling the m_record_screen_dir flag (see line 139 of stella enviornment.cpp) .